cmake/morpheus_utils/python/python_module_tools.cmake

```diff
@@ -367,7 +368,6 @@ function(morpheus_utils_build_python_package PACKAGE_NAME)
 
   # Change which setup we use if we are using inplace
   if(_ARGS_IS_INPLACE)
-    list(APPEND _pip_command "-e")
```

Comment:
Why was this removed? This will cause the files to be copied and leads to some confusion.

Reply:
I was told it was unnecessary 🤷🏻‍♂️